### PR TITLE
[MOB-15255] ACC doc updates

### PR DESCRIPTION
**Pull request by rymorale**
Updates based on the doc guidelines

### CODE REVIEW

**Review by calebk1 — APPROVED** [review]
LGTM!

File: using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-api-reference.md [hunk 1]
```diff
@@ -155,182 +167,198 @@ func application(_ application: UIApplication, didRegisterForRemoteNotifications
 {% endtab %}
 {% endtabs %}
 
-## trackNotificationReceive
+## trackNotificationClick
```
Comment (swarna04) [inline using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-api-reference.md:170]: The tab formatting is off in the current docs - https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-api-reference#tracknotificationclick. Verify this is fixed in your pr.

**Review by swarna04 — CHANGES_REQUESTED** [review]
Let's also cleanup the Configuration keys - https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-campaignclassic#configuration-keys. We do not need env prefixed keys here since Configuration is environment aware.

**Comment by swarna04** [comment]
Provided feedback here @rymorale, let me know if you have any questions.
cc: @calebk1

File: using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-api-reference.md [hunk 2]
```diff
@@ -155,182 +167,198 @@ func application(_ application: UIApplication, didRegisterForRemoteNotifications
...
+{% tab title="iOS (AEP 3.x)" %}
 
-{% hint style="info" %}
-You can pass the `launchOptions` that were received upon opening the application or `userInfo` , which contains the received push payload in `trackInfo`. If `trackInfo` is null or does not contain the necessary tracking identifiers, `broadlogId` (`_mId`) and `deliveryId` (`_dId`), a track request is **not** sent.
+This extension is not compatible with the AEPCore extension at this time, for more details see [Current SDK Versions](https://aep-sdks.gitbook.io/docs/resources/upgrading-to-aep/current-sdk-versions#ios-swift)
+
+{% endtab %}
+
+{% tab title="iOS (ACP 2.x)" %}
+{% hint style="info"}
```
[the remark this answers is not in the document]
Comment (rymorale) [reply using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-api-reference.md:226]: nice catch

File: using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-api-reference.md (hunk 1 above)
Comment (rymorale) [reply using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-api-reference.md:170]: I believe this should be fixed in this pr.

**Comment by rymorale** [comment]
@swarna04. I made the review updates. please take another look when you get the chance. thanks!

File: using-mobile-extensions/adobe-campaignclassic/README.md [hunk 3]
```diff
@@ -66,7 +66,13 @@ The request timeout is the amount of time, in seconds, to wait for a response fr
    ```
 {% endtab %}
 
-{% tab title="iOS" %}
+{% tab title="iOS (AEP 3.x)" %}
+
+This extension is not compatible with the AEPCore extension at this time, for more details see [Current SDK Versions](https://aep-sdks.gitbook.io/docs/resources/upgrading-to-aep/current-sdk-versions#ios-swift)
```
[the remark this answers is not in the document]
Comment (calebk1) [reply using-mobile-extensions/adobe-campaignclassic/README.md:71]: Doubly agreeing on this!

File: using-mobile-extensions/adobe-campaignclassic/README.md [hunk 4]
```diff
@@ -127,7 +134,13 @@ public class CampaignClassicTestApp extends Application {
 ```
 {% endtab %}
 
-{% tab title="iOS" %}
+{% tab title="iOS (AEP 3.x)" %}
+
+This extension is not compatible with the AEPCore extension at this time, for more details see [Current SDK Versions](https://aep-sdks.gitbook.io/docs/resources/upgrading-to-aep/current-sdk-versions#ios-swift)
```
Comment (calebk1) [inline using-mobile-extensions/adobe-campaignclassic/README.md:139]: Same comment here - update this to a relative link instead.

File: using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-api-reference.md [hunk 5]
```diff
@@ -155,160 +168,182 @@ func application(_ application: UIApplication, didRegisterForRemoteNotifications
...
 
 {% tabs %}
 {% tab title="Android" %}
+
 ### Java
 
-{% hint style="info" %}
-If `trackInfo` is null, or does not contain the necessary tracking identifiers, `messageId` (`_mId`) and `deliveryId` (`_dId`), a track request is **not** sent.
-{% endhint %}
+If `trackInfo` is null or does not contain the necessary tracking identifiers, `messageId` (`_mId`) and `deliveryId` (`_dId`), a track request is **not** sent.
```
Comment (calebk1) [inline using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-api-reference.md:283]: Is there a reason we aren't wrapping this in a hint block?

**Review by calebk1 — COMMENTED** [review]
Just a few small comments left!

File: using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-api-reference.md [hunk 6]
```diff
@@ -23,7 +23,7 @@ String campaignClassicExtensionVersion = CampaignClassic.extensionVersion();
 
 {% tab title="iOS (AEP 3.x)" %}
 
-This extension is not compatible with the AEPCore extension at this time, for more details see [Current SDK Versions](https://aep-sdks.gitbook.io/docs/resources/upgrading-to-aep/current-sdk-versions#ios-swift)
+This extension is not compatible with the AEPCore extension at this time, for more details see [Current SDK Versions](../../resources/upgrading-to-aep/current-sdk-versions#ios-swift)
```
Comment (swarna04) [inline using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-api-reference.md:26]: Should this be `../../resources/upgrading-to-aep/current-sdk-versions.md#ios-swift`?

File: using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-api-reference.md [hunk 7]
```diff
@@ -333,7 +334,17 @@ func trackNotificationReceive(_ trackInfo: [String: String])
...
+  if launchOptions != nil {
+    print("launchOptions: \(launchOptions.description)")
+  }
+  // Tracking silent push notification receive
+  if (launchOptions["aps"]["content-available"] as? NSNumber).intValue == 1 {
+    print("Silent Push Notification")
+    ACPCampaignClassic.trackNotificationReceive(launchOptions)
+    completionHandler(UIBackgroundFetchResultNoData)
+  }
+}
```
Comment (swarna04) [inline using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-api-reference.md:347]: Suggesting minor edits here
```
func application(_ application: UIApplication,
didReceiveRemoteNotification userInfo: [AnyHashable : Any],
                 fetchCompletionHandler completionHandler: @escaping (UIBackgroundFetchResult) -> Void) {
    
    guard let aps = userInfo["aps"] as? [String: Any] else {
        completionHandler(.failed)
        return
    }
    if aps["content-available"] as? Int == 1 {
        // Track silent push notification receive
        ACPCampaignClassic.trackNotificationReceive(userInfo)
        completionHandler(.noData)
    }
 }
```

File: using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-api-reference.md [hunk 8]
```diff
@@ -333,7 +334,17 @@ func trackNotificationReceive(_ trackInfo: [String: String])
 **Example**
 
 ```swift
-ACPCampaignClassic.trackNotificationReceive(trackInfo[String:String])
+func application(_ application: UIApplication, didReceiveRemoteNotification launchOptions: [AnyHashable : Any], fetchCompletionHandler completionHandler: @escaping (UIBackgroundFetchResult) -> Void) {
+  if launchOptions != nil {
```
Comment (swarna04) [inline using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-api-reference.md:338]: You will mostly receive a warning for this nil check as `launchOptions` here is non-optional.

**Review by swarna04 — COMMENTED** [review]
Left a couple of minor comments here @rymorale , should be good to merge once we resolve those.

File: using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-api-reference.md (hunk 5 above)
Comment (rymorale) [reply using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-api-reference.md:283]: no, but it should be. i'll get it in a hint block.

File: using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-api-reference.md (hunk 6 above)
Comment (rymorale) [reply using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-api-reference.md]: updated the message regarding AEP 3.x campaign classic. The linked page didn't give much info.

File: using-mobile-extensions/adobe-campaignclassic/README.md [hunk 9]
```diff
@@ -68,7 +68,7 @@ The request timeout is the amount of time, in seconds, to wait for a response fr
 
 {% tab title="iOS (AEP 3.x)" %}
 
-This extension is not compatible with the AEPCore extension at this time, for more details see [Current SDK Versions](https://aep-sdks.gitbook.io/docs/resources/upgrading-to-aep/current-sdk-versions#ios-swift)
+Adobe Campaign Classic has not yet been released as an AEP 3.x Swift extension. Please reach out to your Adobe customer account manager if you have any questions or would like to express interest in the AEP 3.x Campaign Classic extension.
```
Comment (swarna04) [inline using-mobile-extensions/adobe-campaignclassic/README.md:71]: @calebk1 Ryan's updated the verbiage for the lack of ACC 3.x Swift extension support. Can you please review this and let us know of any changes needed here. Ty.

**Review by swarna04 — APPROVED** [review]

File: using-mobile-extensions/adobe-campaignclassic/README.md (hunk 9 above)
Comment (calebk1) [reply using-mobile-extensions/adobe-campaignclassic/README.md:71]: Looks good to me!

**Pull request merged**